docs/source/2018Michael.rst

```diff
@@ -57,10 +57,10 @@ Hurricane Track and Intensity
   :width: 400
   :align: center
 
-  Hurricane tracks from GFSv16beta (red dashline), GFSv15p2 (blue dash line), and BestTrack (black dash line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
+  Hurricane tracks from GFSv16beta (red dash line), GFSv15p2 (blue dash line), and BestTrack (black dash line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
```

Comment:
I think it is "dashed" line - but please double check.

Reply:
Done

Reply:
I still think it is "dashed" and not "dash" ;-)